templates/vue-demo-store/pages/account/address.vue

```diff
@@ -72,7 +79,12 @@ onBeforeMount(async () => {
       {{ $t("account.addressAddNew") }}
     </button>
     <SharedModal :controller="addAddressModalController">
-      <SharedAccountAddressForm @success="addAddressModalController.close" />
+      <SharedAccountAddressForm
+        @success="
+          async () =>
```

Comment:
Maybe adding it as a function in the script would be better? Only for a better readable. What do you think?

Reply:
I had it like that before but then I was thinking why a new function that uses some already existing functions and then I find it better to put it on the spot where the function is executed.